.travis.yml

```diff
@@ -1,4 +1,3 @@
-# Copyright 2017 The Forseti Security Authors. All rights reserved.
 #
```

Comment:
Was there a reason you removed the Copyright line??
e.g.
# Copyright 2019 The Forseti Security Authors. All rights reserved.

Reply:
Nope, I intended to modify only the contents. Added it back.